lib/ActivityStreams.js

```diff
@@ -96,25 +96,26 @@ ActivityStreams.prototype = {
    */
   _respondToPlacesRequests(msgName, params) {
     let {msg, worker} = params;
+    const append = msg.meta && msg.meta.append;
```

Comment:
Actually, I remove my comment. `meta` is not sent as a parameter, so query caching will work correctly